src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_content.html.twig

```diff
@@ -17,20 +22,29 @@
                     {% endif %}
                     {{ 'sylius.ui.subscribed_to_newsletter'|trans }}
                 </div>
+                <div id="email-verified">
+                    {% if customer.user.verified %}
```

Comment:
Can we make this `if` inline? `{{ customer.user.verified ? 'green checkmark' : 'red remove' }}`